packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -306,10 +329,10 @@ export function buildRootObject() {
         /** @type {PromiseRecord<ZoeSeatAdmin>} */
         const zoeSeatAdminPromiseKit = makePromiseKit();
         // Don't trigger Node.js's UnhandledPromiseRejectionWarning
-        zoeSeatAdminPromiseKit.promise.catch(_ => {});
+        zoeSeatAdminPromiseKit.promise.catch(_ => { });
         const userSeatPromiseKit = makePromiseKit();
         // Don't trigger Node.js's UnhandledPromiseRejectionWarning
-        userSeatPromiseKit.promise.catch(_ => {});
+        userSeatPromiseKit.promise.catch(_ => { });
         const seatHandle = makeHandle('SeatHandle');
```

Comment:
why this change?

Reply:
That was from running the wrong lint-reformatter. I'll re-run `lint-fix`